Detectors/ITSMFT/ITS/reconstruction/src/CAaux.cxx

```diff
@@ -43,7 +33,7 @@ Track::Track(float x, float a, float* p, float* c, int *cl) :
   }
 
 bool Track::Update(const Cluster *cl) {
-  float p[2] = {cl->y,cl->z};
+  array<float,2> p{cl->y,cl->z};
   if (!mT.Update(p,cl->cov)) return false;
```

Comment:
@shahor02 Thanks for pointing out the bug twice!
It was there since the beginning of the porting, but it is not present in AliRoot since there I am using Yura's code.
As of the std::array usage: as Mikolaj suggested I changed the interfaces of functions to use std::array instead of C-style array to have a stricter type check. Since it does not add any runtime overhead and in this case it does not mess up the code, I changed it.